bot/exts/moderation/infraction/_utils.py

```diff
@@ -31,12 +31,13 @@
 Infraction = t.Dict[str, t.Union[str, int, bool]]
 
 APPEAL_SERVER_INVITE = "https://discord.gg/WXrCJxWBnm"
+MODMAIL_ACCOUNT_ID = "683001325440860340"
 
 INFRACTION_TITLE = "Please review our rules"
 INFRACTION_APPEAL_SERVER_FOOTER = f"\nTo appeal this infraction, join our [appeals server]({APPEAL_SERVER_INVITE})."
 INFRACTION_APPEAL_MODMAIL_FOOTER = (
     '\nIf you would like to discuss or appeal this infraction, '
-    'send a message to the ModMail bot.'
+    f'send a message to <@{MODMAIL_ACCOUNT_ID}>.'
```

Comment:
What is the difference when having `<@{MODMAIL_ACCOUNT_ID}>` opposed to `<@!{MODMAIL_ACCOUNT_ID}>`?

Reply:
Since this is sent in a DM channel, the bot doesn't have a nickname, so the ! will do nothing right?

Reply:
![image](https://user-images.githubusercontent.com/32915757/210251915-48eb61f4-3098-46d1-877a-5bf627a44e77.png)

This is how `<@!683001325440860340>` rendered for me in my DMs with the Python bot. But <@683001325440860340>` renders the same way.